tpgtools/overrides/compute/beta/network_firewall_policy.yaml

```diff
@@ -0,0 +1,17 @@
+- type: CUSTOM_RESOURCE_NAME
+  details:
+    title: regional_network_firewall_policy
```

Comment:
nit: rather than `regional`, we should prefer `region`. That's consistent with the rest of GCE resources, and the naming in the API: https://cloud.google.com/compute/docs/reference/rest/v1/regionNetworkFirewallPolicies

Reply:
agreed and fixed